x-pack/legacy/plugins/uptime/index.ts

```diff
@@ -33,6 +34,13 @@ export const uptime = (kibana: any) =>
       home: ['plugins/uptime/register_feature'],
     },
     init(server: KibanaServer) {
-      initServerWithKibana(server);
+      const initializerContext = {} as PluginInitializerContext;
+      const { elasticsearch, xpack_main } = server.plugins;
+      plugin(initializerContext).setup({
```

Comment:
The signature for setup is `setup(core, plugins)` with all plugin dependencies being passed in as the second argument. Usage collector will be exposed from the telemetry plugin and xpack licencing is available as a NP plugin. So it would be a bit more accurate to move these dependencies into a second argument.